python/create_service_diagnostics_bundle.sh

```diff
@@ -5,18 +5,14 @@ set -eu -o pipefail
 readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly DCOS_SERVICE_DIAGNOSTICS_SCRIPT_PATH="dcos-sdk-service-diagnostics/python"
 
+#ToDo: the image version should be externally managed outside this file.
```

Comment:
Um, what I meant this comment should do is provide a hint to the person who changes it next time that they should go go to `data-services-ci` and change the value there to match.
And now that I think about this, this might not really work, unless we serialize these PRs - since there is a race window where another person's PRs will be tested with the wrong version.
So we might need to wait for #16 after all.